src/VisualStudio/Core/Def/Implementation/UnusedReferences/VisualStudioReferenceCleanupService.cs

```diff
@@ -20,13 +20,13 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.UnusedReference
     [ExportWorkspaceService(typeof(IReferenceCleanupService), ServiceLayer.Host), Shared]
     internal sealed class VisualStudioReferenceCleanupService : IReferenceCleanupService
     {
-        private readonly IProjectSystemReferenceCleanupService _projectSystemReferenceUpdateService;
+        private readonly IProjectSystemReferenceCleanupService2 _projectSystemReferenceUpdateService;
 
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
         public VisualStudioReferenceCleanupService(IProjectSystemReferenceCleanupService projectSystemReferenceUpdateService)
         {
-            _projectSystemReferenceUpdateService = projectSystemReferenceUpdateService;
+            _projectSystemReferenceUpdateService = (IProjectSystemReferenceCleanupService2)projectSystemReferenceUpdateService;
```

Comment:
With this change It seems that we don't need pr anymore https://github.com/dotnet/project-system/pull/7425